test/nidmExampleDataTest.m

```diff
@@ -1,10 +1,46 @@
+%To run the below tests use the following code:
+%import matlab.unittest.TestSuite;
+%tests = matlab.unittest.TestSuite.fromFile(which('nidmExampleDataTest'))
+%result = run(tests)
+
 classdef nidmExampleDataTest < matlab.unittest.TestCase
+    
+    methods(TestMethodSetup)
+        %Rename the users HTML folder to prevent the tests damaging it.
+        function storeUsersHTML(testCase)
+            if exist(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html')) == 7 
+                movefile(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html'), strrep(fileparts(mfilename('fullpath')), 'test', 'Data\htmlTemp'))
+            end
+        end
+    end
+    
+    methods(TestMethodTeardown)
+        %Remove the HTML folder created by test.
+        function removeTestHTML(testCase)
+            pause(0.1);
+            if exist(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html')) == 7
+                rmdir(strrep(fileparts(mfilename('fullpath')), 'test', 'Data\html'), 's')
+            end
+        end
+        %Move the users data back to the HTML folder.
+        function addUsersHTML(testCase)
```

Comment:
Could this be in the same function as `removeTestHTML`? I might be wrong but I don't think Matlab is running all those tear down functions in a specific order which means that `removeTestHTML` might be run after `addUsersHTML`.